satpy/etc/composites/seviri.yaml

```diff
@@ -193,12 +193,12 @@ composites:
     - ctth_tempe_pal
     standard_name: cloud_top_temperature
 
-  cloud_top_phase:
-    compositor: !!python/name:satpy.composites.PaletteCompositor
-    prerequisites:
-    - cmic_phase
-    - cmic_phase_pal
-    standard_name: cloud_top_phase
+  # cloud_top_phase:
+  #   compositor: !!python/name:satpy.composites.PaletteCompositor
+  #   prerequisites:
+  #   - cmic_phase
+  #   - cmic_phase_pal
+  #   standard_name: cloud_top_phase
```

Comment:
Should this remain as a comment or rather be removed from `seviri.yaml` algother?  Perhaps as an alternative just add a comment that this composite was moved to `visir.yaml`?

Reply:
@gerritholl Thanks for reminding me. I reverted back to how it was before here. This belongs to another separate PR! To put all the (NWCSAF) cloud composites in the visir, and that way making them available for all instruments. You are welcome doing that if you like?